lib/octokit/client.rb

```diff
@@ -163,10 +163,10 @@ def inspect
       inspected.gsub! @bearer_token, '********' if @bearer_token
       # Only show last 4 of token, secret
       if @access_token
-        inspected.gsub! @access_token, "#{'*' * 36}#{@access_token[36..-1]}"
+        inspected.gsub! @access_token, "#{'*' * 36}#{@access_token[36..]}"
```

Comment:
Do you mind explaining this change to me? Were we only showing the first, second, and third from last characters in the token?

Reply:
`String#[s..-1]` and `String#[s..]` are compatible with the endless range syntax added in Ruby 2.6.